docs/WARP-GRAPH.md

```
@@ -0,0 +1,1214 @@
+impl WarpId { pub fn as_bytes(&self) -> &Hash { &self.0 } }
+```
+
+No behavior changes, just safer ABI expectations.
+
+⸻
+
+2. Add a wsc module (where it belongs)
+
+Put this in warp-core (or whatever crate defines GraphStore/AttachmentValue), e.g.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)